1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,153 @@
+        And User has the "<Role>" role in the project's scope
+        When they click the Add Project Manager Button
+        Then they see fields:
+            | Start Date  |
+            | User search |
+        Examples:
+            | Role            |
+            | Curator         |
+            | Project Owner   |
+            | Project Manager |
+
+    @xxxb
+    Scenario: User adds a Project Manager
+        Given User opens Dialog for adding Project Manager
+        When they select a Start Date
+        And they enter a name in the User search field
+        And they select a User from the search
+        And they click the Add Button
+        Then they see the User listed as a Project Manager
+
+
+    Scenario Outline: User opens Dialog for adding Project Participant
+        Given User opens Participants tab for Project
+        And User has the "<Role>" role in the project's scope
+        When they click the Add Project Participant Button
+        Then they can select role to be either of:
+            | Project Partcipant    |
+            | Local Project Manager |
+        And they can see User search field
+        Examples:
+            | Role                  |
+            | Curator               |
+            | Project Owner         |
+            | Project Manager       |
+            | Local Project Manager |
+
+    @xxx
+    Scenario: User adds a Project Participant
+        Given User opens Dialog for adding Project Participant
+        When they enter a name in the User search field
+        And they select a role
+        And they select a User from the search
+        And they click the Add Button
+        Then they see the User listed as a Project Participant
+
+    @xxx
+    Scenario Outline: User opens Financing tab for Project
+        Given User sees the Project Wizard
+        And User has the "<Role>" role in the project's scope
+        When they click the Financing tab
+        Then they see fields for:
+            | Financing Code |
+        And they see a Previous Button
+        And they see a Support Button
+        And they see a Publish Button
+        Examples:
+            | Role            |
+            | Curator         |
+            | Project Owner   |
+            | Project Manager |
+
+    @xxx
+    Scenario: User adds a Financing source for Project
+        Given User opens Financing tab for Project
+        When they enter a search text in the search field
+        And they select a Financing source from the search results
+        Then they see that the Financing Code is added to the project
+        And they see a Previous Button
+        And they see a Support Button
+        And they see a Publish Button
+
+    @xxx
+    # Vi må unngå at ein bruker trykker Save på eit publisert prosjekt og trur at han har publisert endringer på prosjektet. Skal vi endre/fjerne Save knapp, når prosjektet er publisert - eller endre logikken til eit publisert prosjekt?
```

Comment:
Betyr dette at vi egentlig vil ha to versjoner av et prosjekt i dette tilfellet?
En publisert versjon som vises på Landing Page, og en draft versjon som bare Owner kan se, helt til Owner publiserer som beskrevet i scenario under, og da blir det bare en publisert versjon igjen?

Reply:
Har prøvd å skrive meg rundt denne kompleksiteten. Håper å unngå den.